docs/changes/stats/11664.feature.rst

```diff
@@ -0,0 +1,3 @@
+Added the ability to specify stdfunc='mad_std' when doing sigma clipping,
+which leads to significant performance improvements if cenfunc is 'mean'
```

Comment:
maybe "which will use a built-in function and give significant ..."

Reply:
Fixed